public/mission/index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+  <link rel="manifest" href="/site.webmanifest" />
+  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#BA4B2B" />
+  <meta name="msapplication-TileColor" content="#2b5797" />
+  <meta name="theme-color" content="#ffffff" />
+  <meta name="description" content="Magmo | Bringing scalability to the decentralized web" />
+
+  <meta property="og:description" content="Magmo | Bringing scalability to the decentralized web" />
+  <meta property="og:site_name" content="Magmo" />
+  <meta property="og:title" content="Magmo" />
+  <meta property="og:url" content="https://magmo.com/" />
+  <meta property="og:type" content="website" />
+  <meta property="og:image" content="https://magmo.com/assets/symbol.png" />
+  <meta property="og:image:width" content="400" />
+  <meta property="og:image:height" content="400" />
+
+  <meta name="twitter:title" content="Magmo | Bringing scalability to the decentralized web" />
+  <meta name="twitter:description" content="Magmo | Bringing scalability to the decentralized web" />
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:url" content="https://magmo.com/" />
+  <meta name="twitter:image" content="https://magmo.com/assets/symbol.png" />
+
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-114409337-6"></script>
```

Comment:
It's quite possible that this doesn't belong here, given how this site was put together.